### DISCUSSION TITLE
Comparison with paperclip & okapi

### DISCUSSION THREAD

**Discussion by jayvdb** [juhaku/utoipa#592] [General]
It would be useful to collate and provide and comparison with https://github.com/paperclip-rs/paperclip , especially features of each that are missing in the other.  This will help users pick the right one for their needs, and also identify areas that each can improve so that the Rust community has two great options for this problem.

**Comment by juhaku** [comment]
Yeah, personally I don't know enough about the paperclip to be able to do that. But I am for this, if we can find enough people collaborate on this matter. I am planning to gather a list of features the `utoipa` is coming with in near future after I have finished few other issues first.

**Comment by jayvdb** [comment]
Another one to compare it with : https://github.com/GREsau/okapi
